src/main/java/com/iota/iri/network/pipeline/ReceivedStage.java

```diff
@@ -21,20 +24,25 @@
     private TransactionRequester transactionRequester;
     private TransactionSolidifier txSolidifier;
     private SnapshotProvider snapshotProvider;
+    private MilestoneService milestoneService;
+    private Hash cooAddress;
 
     /**
      * Creates a new {@link ReceivedStage}.
-     *
+     * 
      * @param tangle           The {@link Tangle} database used to store/update the transaction
-     * @param txSolidifier      The {@link TransactionSolidifier} used to store/update the transaction
+     * @param txSolidifier      The {@link TransactionValidator} used to store/update the transaction
```

Comment:
huh?

Reply:
Must have happened during the conflict resolution while merging dev in, i'll touch that up.